kde-apps/spectacle/spectacle-23.04.3.ebuild

```diff
@@ -55,7 +55,7 @@ DEPEND="${COMMON_DEPEND}
 "
 RDEPEND="${COMMON_DEPEND}
 	>=dev-qt/qdbus-${QTMIN}:*
-	>=dev-qt/qtmultimedia-${QTMIN}:5[qml]
+	video-preview? ( >=dev-qt/qtmultimedia-${QTMIN}:5[qml] )
```

Comment:
This introduces a pure runtime optional USE flag which we don't do - but I'm a bit confused because you seemed to clock that in your gwenview optfeature change.

See https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001.